packages/scripts/import/convertJsonRowToEntryFormat.ts

```diff
@@ -11,7 +11,7 @@ export function convertJsonRowToEntryFormat(
   Boolean(row.phonetic) && (entry.ph = row.phonetic);
   Boolean(row.morphology) && (entry.mr = row.morphology);
   Boolean(row.interlinearization) && (entry.in = row.interlinearization);
-  Boolean(row.partOfSpeech) && (entry.ps = row.partOfSpeech);
+  Boolean(row.partOfSpeech) && (entry.ps = row.partOfSpeech.split(', '));
```

Comment:
@Danble Add a test case for "n,v" without the space and also "n,  v" because you're guaranteed to have users not use a space or use two (e.g. Chinese often don't notice when spaces are missing or doubled when writing English because they don't use spaces in their writing system). To solve those test cases will then require you to split just the comma and trim each item.

Reply:
Thanks!